### PR TITLE
CEXT-2107: Create a CLI command to show event registrations

### PR DESCRIPTION
**Pull request by oshmyheliuk**
## Purpose of this pull request

This pull request (PR) adds information about the new CLI command to show a list of event registrations

## Affected pages

- https://developer.adobe.com/commerce/events/get-started/commands/#get-details-about-a-configured-event-provider

## Links to Magento Open Source code

- ...

### CODE REVIEW

File: src/pages/get-started/commands.md [hunk 1]
```diff
@@ -88,6 +88,44 @@ Configured event provider details:
...
+bin/magento events:registrations:list
+```
+
+### Response
+
+```terminal
+- Event registration 1 (5eb5106d-0e84-4c2e-ae73-b22b266a7908)
+```
+
+To include information about subscribed events increase the verbosity of the command
```
Comment (keharper) [inline src/pages/get-started/commands.md:111]:
```suggestion
Specify the `-vv` option to increase the verbosity of information about the subscribed events.
```

File: src/pages/get-started/commands.md [hunk 2]
```diff
@@ -88,6 +88,44 @@ Configured event provider details:
 - description: testing local provider creation (Instance docs-stage-testing)
 ```
 
+## Get details about configured event registrations in your App Builder application
```
Comment (keharper) [inline src/pages/get-started/commands.md:91]: Add a link to this section at about line 23 of this file.

**Comment by keharper** [comment]
One more thing: This needs to be rebased to the `develop` branch.

File: src/pages/get-started/commands.md (hunk 2 above)
Comment (oshmyheliuk) [reply src/pages/get-started/commands.md:92]: Added

**Pull request merged**